sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -932,10 +998,14 @@ def get_buffer(buffer_id):
       return pcoll_buffers[buffer_id]
 
     def get_input_coder_impl(transform_id):
-      return context.coders[
-          safe_coders[beam_fn_api_pb2.RemoteGrpcPort.FromString(
-              process_bundle_descriptor.transforms[transform_id].spec.payload).
-                      coder_id]].get_impl()
+      coder_id = beam_fn_api_pb2.RemoteGrpcPort.FromString(
+          process_bundle_descriptor.transforms[transform_id].spec.payload).\
```

Comment:
Lint: don't use backslashes for continuation. (These days you can just run yapf to format your code, see https://cwiki.apache.org/confluence/display/BEAM/Python+Tips.)